n3fit/src/evolven3fit_new/evolve.py

```diff
@@ -165,16 +165,16 @@ def evolve_exportgrid(exportgrid, eko, x_grid):
     # evolve pdf
     evolved_pdf = apply.apply_pdf(eko, pdf_to_evolve)
     # generate block to dump
-    targetgrid = list(eko["targetgrid"])
+    targetgrid = eko.rotations.targetgrid.tolist()
 
     def ev_pdf(pid, x, Q2):
         return x * evolved_pdf[Q2]["pdfs"][pid][targetgrid.index(x)]
 
     block = genpdf.generate_block(
         ev_pdf,
         xgrid=targetgrid,
-        Q2grid=list(eko["Q2grid"]),
-        pids=br.flavor_basis_pids,
+        Q2grid=list(eko.mu2grid),
```

Comment:
Later on I would consider:
```suggestion
        mu2grid=list(eko.mu2grid),
```

For consistency with:
- new EKO notation
- physical meaning
- Python naming (PEP 8)

But of course this would follow from the matching change in the function in `genpdf`. But actually that is already done in a PR https://github.com/NNPDF/eko/pull/220, so consider this more as a warning for the future...